homeassistant/components/freebox/config_flow.py

```diff
@@ -105,3 +105,9 @@ async def async_step_link(self, user_input=None):
     async def async_step_import(self, user_input=None):
         """Import a config entry."""
         return await self.async_step_user(user_input)
+
+    async def async_step_zeroconf(self, discovery_info: dict):
+        """Initialize flow from zeroconf."""
+        host = discovery_info.get("properties", {}).get("api_domain")
+        port = discovery_info.get("properties", {}).get("https_port")
+        return await self.async_step_user({CONF_HOST: host, CONF_PORT: port})
```

Comment:
We need to handle `None` for host and port.

Reply:
Actually those properties are always here